### DISCUSSION TITLE
Add steam cloud support

### DISCUSSION THREAD

**Discussion by Overlordtom** [voxeltycoon/issues#530] [General] [answered]
I play often at home and on the road with a laptop, it would be nice to have the two synced together via steam cloud

**Comment by Tordolini** [comment]
It's friday night, I have my Rush mix tape in my casette deck and as this issue has been moved to discussion I would really like to discuss this issue.

As far as I can see, my four save games is about the same size as the game itself. If the save games are too spacy for Steam cloud, would there be a way to change the (manual) save game location? If Steam is stingy with cloud storage I am happy to provide my own plethora of clouds.  Not to brag, but I am willing to do roaming profile, or even zip/rar archive spanning across multiple 3,5'' disks if that is required. As long as I may set the save destination. And yes, I'd probably cheat and set it to something cloud and corrupt all my saves, but at least that would be my prerogative. Mmmm.... prerogative 🤤

**Reply by andrewpey** [reply]
While there's no UI yet for changing saves location, you can [do this by `Settings.json` modification](https://github.com/voxeltycoon/issues/discussions/519#discussioncomment-677147).

**Comment by andrewpey** [comment]
This is planned.

**Comment by andrewpey** [comment, marked as answer]
The latest build on the [beta branch](https://voxeltycoon.xyz/devlog/beta-branch/) has introduced the Steam Cloud saves support.

**Reply by IDragonfire** [reply]
Very good feature!
